src/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -109,9 +109,19 @@
     @Option(doc = "Read two barcode SAM tag (ex. BX for 10X Genomics)", optional = true)
     public String READ_TWO_BARCODE_TAG = null;
 
+    @Option(doc = "If true remove optical duplicates from the output even if REMOVE_DUPLICATES is false.")
+    public boolean REMOVE_OPTICAL_DUPLICATES = false;
```

Comment:
Might want to mention that if REMOVE_DUPLICATES is set to true then it overrides this being set to false?